test/executor.test.js

```diff
@@ -450,6 +450,38 @@ describe('executor', function() {
       });
     });
 
+    it('should parse simple config variable from env var', function(done) {
+      process.env.restApiRoot = '/url-from-env-var';
+      boot.execute(app, simpleMiddlewareConfig('routes',
+        { path: '${restApiRoot}' }
+      ));
+
+      supertest(app).get('/url-from-env-var').end(function(err, res) {
+        if (err) return done(err);
+        expect(res.body.path).to.equal('/url-from-env-var');
+        delete process.env.restApiRoot;
+        done();
+      });
+    });
+
+    it('dynamic variable from `env var` should have' +
+    ' precedence over app.get()', function(done) {
+      process.env.restApiRoot = '/url-from-env-var';
```

Comment:
this will leave `process.env` polluted for other unit-tests, which may lead to test failure that will be difficult to debug.

Please add a `beforeEach` hook to reset `restApiRoot` to undefined value before any test is run.